src/operator/quantization/quantized_reshape.cc

```diff
@@ -0,0 +1,120 @@
+          [](const NodeAttrs& attrs) {                                                       \
+            return std::vector<std::pair<int, int> >{{0, 0}, {1, 1}, {2, 2}};                \
+          })                                                                                 \
+      .set_attr<FCompute>("FCompute<cpu>", QuantizedReshapeCompute)                          \
+      .set_attr<FResourceRequest>(                                                           \
+          "FResourceRequest",                                                                \
+          [](const NodeAttrs& n) {                                                           \
+            return std::vector<ResourceRequest>{ResourceRequest::kTempSpace};                \
+          })                                                                                 \
+      .set_attr<nnvm::FInferType>("FInferType", QuantizedReshapeType)                        \
+      .set_attr<nnvm::FGradient>("FGradient", MakeZeroGradNodes)                             \
+      .set_attr<FQuantizable>("FQuantizable",                                                \
+                              [](const NodeAttrs& attrs) { return QuantizeType::kSupport; }) \
+      .add_argument("data", "NDArray-or-Symbol", "Array to be reshaped.")                    \
+      .add_argument("min_data",                                                              \
+                    "NDArray-or-Symbol",                                                     \
+                    "The minimum scalar value "                                              \
+                    "possibly produced for the data")                                        \
+      .add_argument("max_data",                                                              \
+                    "NDArray-or-Symbol",                                                     \
+                    "The maximum scalar value "                                              \
+                    "possibly produced for the data")
+
+MXNET_OPERATOR_REGISTER_QUANTIZED_RESHAPE(_contrib_quantized_reshape)
+    .add_alias("quantized_reshape")
+    .set_attr_parser(ParamParser<ReshapeParam>)
+    .set_attr<mxnet::FInferShape>("FInferShape", QuantizedReshapeInferShape<ReshapeShape>)
+    .add_arguments(ReshapeParam::__FIELDS__());
+
+MXNET_OPERATOR_REGISTER_QUANTIZED_RESHAPE(_npx_quantized_reshape)
+    .set_attr_parser(ParamParser<NumpyXReshapeParam>)
+    .set_attr<mxnet::FInferShape>("FInferShape", QuantizedReshapeInferShape<NumpyXReshapeShape>)
+    .add_arguments(NumpyXReshapeParam::__FIELDS__());
+
+template <bool is_numpy_op>
+nnvm::ObjectPtr QuantizedReshapeNode(const NodeAttrs& attrs) {
+  nnvm::ObjectPtr node = nnvm::Node::Create();
+
+  if constexpr (is_numpy_op) {
+    node->attrs.op = Op::Get("_npx_quantized_reshape");
+  } else {
+    node->attrs.op = Op::Get("_contrib_quantized_reshape");
+  }
```

Comment:
You could make the op name a string template parameter instead

Reply:
We could do it using global strings however global variables are not advisable. Instead, after consulting with @bgawrych and @mozga-intel, enumerate version of this functionality was created.